src/aurora.ts

```diff
@@ -184,6 +184,23 @@ export interface AuroraProps {
    * @default {subnetType:aws_ec2.SubnetType.PRIVATE_WITH_EGRESS} - all private subnets
    */
   readonly vpcSubnets?: aws_ec2.SubnetSelection;
+
+  /**
+   * Additional parameters to pass to the database engine
+   *
+   * @default - No parameter group.
+   */
```

Comment:
This docstring needs to also note
   * You can only specify parameterGroup or parameters but not both.